Users/Install.rst

```diff
@@ -57,6 +57,9 @@ system-wide, use:
 
 .. note::
 
+    **Ubuntu Users** - Don't use ``apt-get install coala``
+    for installing coala as that's a different software.
```

Comment:
"as that's a different software." is bad English

use

"as that is different software."